From: Li Zhang <lizhang@suse.de>
Date: Thu, 24 Mar 2022 11:47:08 +0100
Subject: Fix the module building problem for s390

When building the testsuit for s390, it reports errors as the following:

hw-display-virtio-gpu.so: undefined symbol: virtio_vmstate_info
hw-display-virtio-gpu-gl.so: undefined symbol: virtio_gpu_ctrl_response
hw-display-virtio-gpu-pci.so: undefined symbol: virtio_pci_types_register
hw-display-virtio-gpu-pci-gl.so: undefined symbol: virtio_pci_types_register

Signed-off-by: Li Zhang <lizhang@suse.de>
---
 util/module.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/util/module.c b/util/module.c
index 8370d4c6209eda4319342b202f6b..03896b7c469dedc5a80695aa7dee 100644
--- a/util/module.c
+++ b/util/module.c
@@ -125,6 +125,10 @@ bool s390x_blocklist(const char *name)
         "hw-display-qxl",
         "hw-display-virtio-vga",
         "hw-display-virtio-vga-gl",
+        "hw-display-virtio-gpu",
+        "hw-display-virtio-gpu-gl",
+        "hw-display-virtio-gpu-pci",
+        "hw-display-virtio-gpu-pci-gl",
         "hw-usb-host",
         "hw-usb-redirect",
         "hw-usb-smartcard"
